### PR TITLE
Added  workaround for issue - https://projects.theforeman.org/issues/…

### PR DESCRIPTION
**Pull request by Imaanpreet**
…29370

### CODE REVIEW

File: playbooks/katello/katello_nightly_installation.yaml [hunk 1]
```diff
@@ -93,3 +93,20 @@
   - name: configure the katello nightly
     command: foreman-installer --scenario katello --skip-checks-i-know-better --foreman-initial-admin-username {{ sat_user }} --foreman-initial-admin-password {{ sat_pass }} {{ installer_additional_ops }}
     
+  - name: change the 'pool' setting in database.yml file  #workaround for https://projects.theforeman.org/issues/29370
+    replace:
+     path: /etc/foreman/database.yml
+     regexp: '5'
```
Comment (jhutar) [inline playbooks/katello/katello_nightly_installation.yaml:99]: This is too generic regexp. Maybe something like `pool: 5$`?

Comment (Imaanpreet) [reply playbooks/katello/katello_nightly_installation.yaml:99]: firstly, I did like this 'pool: 5$' like and tested. It's removed the entire pool parameter.

Comment (Imaanpreet) [reply playbooks/katello/katello_nightly_installation.yaml]: tested and changed.

**Comment by jhutar** [comment]
Thank you!

**Pull request merged**